src/tools/rbd/action/BenchWrite.cc

```diff
@@ -262,6 +262,11 @@ int execute(const po::variables_map &vm) {
   } else {
     bench_io_size = 4096;
   }
+  if ((bench_io_size > 100*1024*1024) || (bench_io_size < 1))
```

Comment:
For reference to the readahead issue: https://github.com/ceph/ceph/pull/8921/commits/adb14303ccf8a26ed5536bb692ae63d92ac40a9f